src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/casenotes/CaseNotesResource.kt

```diff
@@ -280,6 +293,20 @@ data class CreateCaseNoteRequest(
   val caseNoteText: String,
 )
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
+data class CaseNoteAmendment(
+  @Schema(description = "Free format text body of amendment")
+  val text: String,
+  @Schema(description = "Author login name")
+  val authorUsername: String,
+  @Schema(description = "Author STAFF_ID")
+  val authorUserId: Long?,
```

Comment:
odd that this is `userId`?

Reply:
True, i took this from the dps team spec, but agree, will change it next time